graphql-kaminari_connection.gemspec

```diff
@@ -32,6 +32,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'appraisal', '~> 2.5'
   spec.add_development_dependency 'bundler', '~> 2.2'
   spec.add_development_dependency 'codeclimate-test-reporter', '~> 1.0'
+  spec.add_development_dependency 'concurrent-ruby', '< 1.3.5'
```

Comment:
Why this version constraint added?

Reply:
I made a mistake

I removed the unrelated commits